blk/components/virt.c

```diff
@@ -9,44 +9,37 @@
 #include <sddf/util/printf.h>
 #include <sddf/util/string.h>
 #include <sddf/util/util.h>
+#include <blk_config.h>
 
+/* Uncomment this to enable debug logging */
 // #define DEBUG_BLK_VIRT
 
 #if defined(DEBUG_BLK_VIRT)
 #define LOG_BLK_VIRT(...) do{ sddf_dprintf("BLK_VIRT|INFO: "); sddf_dprintf(__VA_ARGS__); }while(0)
 #else
 #define LOG_BLK_VIRT(...) do{}while(0)
 #endif
-
-
 #define LOG_BLK_VIRT_ERR(...) do{ sddf_dprintf("BLK_VIRT|ERROR: "); sddf_dprintf(__VA_ARGS__); }while(0)
 
-/* TODO: Currently only works for 1 and 2 clients, need to handle multiple clients */
-
-#define MAX_BLK_NUM_CLIENTS 16
 
 #define DRIVER_CH 1
-#define CLIENT_CH_1 3
-#define CLIENT_CH_2 4
+#define CLI_CH_OFFSET 3
```

Comment:
Why is the client channel offset 3 instead of 1? And the driver has a channel of zero?